### DISCUSSION TITLE
Importing a CSV file containing initial labeling of records as relevant or irrelevant, instead of manual labeling

### DISCUSSION THREAD

**Discussion by abedidev** [asreview/asreview#816] [Q&A] [answered]
I used your application on its web platform. Based on my understanding, at the first stages of using your application, the user needs to manually label a number of records/papers as relevant/irrelevant, and then these records will be used as (some kind of) training set for the next steps of automatic screening. I have a question about the first stages of manual labeling. Is it possible to read/upload a CSV file (in your Github code or in the web platform) containing the labeled records as the starting point of the next stages?

**Comment by SagevdBrand** [comment, marked as answer]
Hi @abedicodes ,

Thanks for your question! Yes you can indeed upload partly labeled data, where ASReview will use the part that is labeled will be used as the training data. With partly labeled data I mean that some of the articles in your file have been screened already and received either a 1 (for relevant) or 0 (for irrelevant) articles. These labels (1,0 or nothing for those which you have not seen) should be present in a column called `final_included` or `label_included`. 

More information about preparing datasets: https://asreview.readthedocs.io/en/latest/intro/datasets.html
